eng/Versions.props

```diff
@@ -291,24 +292,21 @@
       create a test insertion in Visual Studio to validate.
     -->
     <NewtonsoftJsonVersion>13.0.1</NewtonsoftJsonVersion>
-    <StreamJsonRpcVersion>2.14.20</StreamJsonRpcVersion>
+    <StreamJsonRpcVersion>2.15.4-alpha</StreamJsonRpcVersion>
```

Comment:
The code comment above warns that updating these may be problematic. Perhaps so, but FWIW the VS repo already has something even newer than this version.